Revise german translation

trkoch opened the pull request:
Proof reading revealed some clumsy translations.

* Change gender to common options (divers is the 3rd gender by law)
* Use or do not translate common loanwords such as tool
* Fix grammar by adjusting to context
* Minor typos

The translation for "Using self-report, you can get (...)" was pretty tough,
but I hope I did not deviate from its original meaning too much.

trkoch committed:
Change link to official information provided by ministry

trkoch committed:
Fix missing translation in privacy statement

trkoch committed:
Revise german translation

Proof reading revealed some clumsy translations.

* Change gender to common options (divers is the 3rd gender by law)
* Use or do not translate common loanwords such as tool
* Fix grammar by adjusting to context
* Minor typos

The translation for "Using self-report, you can get (...)" was pretty tough,
but I hope I did not deviate from its original meaning too much.

michaelmcmillan commented:
Great work! Could you add a config as well? It should be in the `countrySpecific` directory :)

michaelmcmillan commented:
Ups never mind! I see it's already added, sorry about that

trkoch commented:
@michaelmcmillan Yes. Translations, including config, were already added in #528.

trkoch committed:
Change domain to de.coronastatus.eu

Unfortunately coronastatus.de is already taken. As a bonus, the pattern
{country}.coronastatus.eu gives us a fall-back for all european
countries where the domain is not available.

trkoch commented:
Some finishing touches and last minute change of domain. Anything missing?

floste7 commented on app/locales/de-DE.json:
```diff
@@ -53,7 +53,7 @@
-  "Help us uncover and predict the corona spread in your area": "Hilf uns die Corona Verbreitung in deiner Gegend <span>aufzudecken</span> und <span>vorauszusagen</span>",
+  "Help us uncover and predict the corona spread in your area": "Hilf uns die Verbreitung von Corona in deiner Gegend <span>aufzudecken</span> und <span>vorauszusagen</span>",
   "here": "hier",
   "How will my data be used?": "Wir werden meine Daten verwendet?",
```
I think this is supposed to be "**Wie** werden meine Daten verwendet?"
